resolwe_bio/docker_images/common/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/resolwebio/base:ubuntu-18.04
+FROM public.ecr.aws/genialis/resolwebio/base:ubuntu-20.04-03042021
```

Comment:
Is it too late to use "iso standard" date in the form of 20210403? To me, year-month-day feels a bit more normal ordering.